ccpp/data/GFS_typedefs.F90

```diff
@@ -649,6 +649,7 @@ module GFS_typedefs
     logical              :: cplwav2atm      !< default no wav->atm coupling
     logical              :: cplchm          !< default no cplchm collection
     logical              :: use_cice_alb    !< default .false. - i.e. don't use albedo imported from the ice model
+    integer              :: cpl_grid_id     !< default grid 1 (parent)
```

Comment:
I'd like to move this into the nems.configure file and attach it to the FV3 cap attributes instead of in the input.nml file.

Reply:
@danrosen25, I think that's a good idea. Could you please help to make the needed code changes in a feature branch then merge it back into this feature/hafs_nest_cpl branch after testing? Thanks!